h2o-algos/src/main/java/hex/schemas/GBMV3.java

```diff
@@ -84,6 +86,9 @@
     @API(help="Column sample rate (from 0.0 to 1.0)", level = API.Level.critical, gridable = true)
     public double col_sample_rate;
 
+    @API(help = "A mapping representing monotonic constraints. Use +1 to enforce an increasing constraint and -1 to specify a decreasing constraint.", level = API.Level.secondary)
+    public KeyValueV3[] monotone_constraints;
```

Comment:
I wonder how checkpointing will work.

Is it possible to continue "improving" a model with monotonic constraints ? Or even better, remove them selectively by setting them to zero ?

Reply:
Good question. I think people can keep training a model with constraints. It would even be possible to remove the constraints (or define them in the opposite direction - not much sense though). The new trees will respect the new parameters.